### PR TITLE
add new guides

### PR DESCRIPTION
**Pull request by inian**

### CODE REVIEW

File: apps/reference/docs/guides/realtime/broadcast-presence.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+id: broadcast
+title: Broadcast
+description: Getting started with Realtime's broadcast feature
+---
+
+Supabase Broadcast allows a client to broadcast messages and multiple receivers to receive the broadcasted message. The broadcasted messages are emphermal, they are not persisted to the database and are directly relayed by the Realtime server.
```
Comment (w3b6x9) [inline apps/reference/docs/guides/realtime/broadcast-presence.mdx:7]:
```suggestion
Supabase Broadcast allows a client to broadcast messages and multiple receivers to receive the broadcasted message. The broadcasted messages are ephemeral, they are not persisted to the database and are directly relayed by the Realtime server.
```

**Review by w3b6x9 — APPROVED** [review]
lgtm!

File: apps/reference/docs/guides/realtime/broadcast-presence.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+id: broadcast
+title: Broadcast
```
Comment (dannykng) [inline apps/reference/docs/guides/realtime/broadcast-presence.mdx:3]: Can we call this "Supabase Broadcast" for SEO? Can add `sidebar_label: Broadcast` to keep the sidebar label as is.

```suggestion
title: Supabase Broadcast
```

File: apps/reference/docs/guides/realtime/broadcast-presence.mdx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+description: Getting started with Realtime's broadcast feature
+---
+
+Supabase Broadcast allows a client to broadcast messages and multiple receivers to receive the broadcasted message. The broadcasted messages are ephemeral, they are not persisted to the database and are directly relayed by the Realtime server.
+
+To broadcast a message
```
Comment (dannykng) [inline apps/reference/docs/guides/realtime/broadcast-presence.mdx:9]:
```suggestion
To broadcast a message:
```

File: apps/reference/docs/guides/realtime/broadcast-presence.mdx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        payload: { x: 0, y: 0 },
+      })
+
+      console.log(status)
+    }, 1000)
+  }
+})
+```
+
+To receive the broadcasted message, run this script in another tab
```
Comment (dannykng) [inline apps/reference/docs/guides/realtime/broadcast-presence.mdx:45]:
```suggestion
To receive the broadcasted message, run this script in another tab:
```

**Review by dannykng — COMMENTED** [review]
Noted a few minor things but looks good—thanks for adding this!

**Review by dannykng — APPROVED** [review]

File: apps/reference/docs/guides/realtime/broadcast.mdx [hunk 5]
```diff
@@ -0,0 +1,68 @@
+---
+id: broadcast
+title: Broadcast
+description: Getting started with Realtime's broadcast feature
```
Comment (kiwicopple) [inline apps/reference/docs/guides/realtime/broadcast.mdx:4]:
```suggestion
description: Getting started with Realtime's Broadcast feature
```

File: apps/reference/docs/guides/realtime/broadcast.mdx [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+---
+
+Supabase Broadcast allows a client to broadcast messages and multiple receivers to receive the broadcasted message. The broadcasted messages are ephemeral, they are not persisted to the database and are directly relayed by the Realtime server. This is ideal for broadcasting information like mouse moments where latency is important, but need not be persisted.
```
Comment (kiwicopple) [inline apps/reference/docs/guides/realtime/broadcast.mdx:7]:
```suggestion
Supabase Broadcast allows a client to send messages, and multiple receivers to receive the messages. 

The broadcasted messages are ephemeral. They are not persisted to the database and are directly relayed through the Realtime server. This is ideal for sending information like mouse moments where latency is important, but need not be persisted.
```

**Review by kiwicopple — APPROVED** [review]

File: apps/reference/docs/guides/realtime/presence.mdx [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+id: presence
+title: Presence
+description: Getting started with Realtime's presence feature
+---
+
+Supabase Presence stores and synchronize shared state across clients. Presence is very similar to Broadcast, except that the state is “persisted” in the Channel so that new joiners can immediately get the state without waiting for another user to broadcast a message.
+
+The `sync` event is triggered whenever the shared state changes. The `join` event is triggered when a new client joins the channel and `leave` when a client leaves.
+
+Each client can use the `track` method to store an object in the shared state. Each client can only track one object in the shared state. If `track` is called again by the same client, then the new object will overwrite the previously tracked object in the shared state.
```
Comment (w3b6x9) [inline apps/reference/docs/guides/realtime/presence.mdx:11]: @inian 🔥

**Pull request merged**